sources/packages/backend/libs/utilities/src/integration-utils.ts

```diff
@@ -22,3 +23,20 @@ export function getTotalYearsOfStudy(completionYears: string): number {
       return 1;
   }
 }
+
+/**
+ * Replace the line breaks in the given text.
+ * @param data data to replace the line break.
+ * @param options replace options:
+ * - `replaceText`: text to replace the line break.
+ * @returns line break replaced string.
+ */
+export function replaceLineBreaks(
```

Comment:
Depending on the direction from https://github.com/bcgov/SIMS/pull/2596/files#r1427358139 we may need to have it at a more generic level if we will sanitize the data before saving it for some cases.